build_runner/lib/src/util/util.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:build/build.dart';
```

Comment:
I'm not a fan of `util.dart` (I'm fine with a directory named `util` though). Maybe `build_dirs.dart`?